scripts/sve-vl.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+### TODO: What is the Windows path for this? (msys2)
```

Comment:
we can just return `scalable` if the file isn't found (and for windows) for now?

Reply:
No, we deal with small layers so it's necessary to know which register size we're going to use. Otherwise it's very difficult to maintain and even the performance will be much worse than using VLST.

Reply:
It's better to ask someone that has Windows on Arm, but for now I changed Makefile so that users can set SVE_VL manually.